src/components/Sheet/index.tsx

```diff
@@ -1,18 +1,19 @@
 import { useTranslation } from 'react-i18next'
+import { useMemo } from 'react'
 import { SheetPanel, SheetPointPanel, SheetItem } from './styled'
-import { createGlobalState, createGlobalStateSetter, useGlobalState } from '../../utils/state'
+import { useBlockchainAlerts, useNetworkErrMsgs } from '../../services/ExplorerService'
 
-const globalNetworkErrMsgs = createGlobalState<string[]>([])
-const globalChainAlerts = createGlobalState<string[]>([])
-
-export const setNetworkErrMsgs = createGlobalStateSetter(globalNetworkErrMsgs)
-export const setChainAlerts = createGlobalStateSetter(globalChainAlerts)
+const ALERT_TO_FILTER_OUT = 'CKB v0.105.* have bugs. Please upgrade to the latest version.'
```

Comment:
I think maybe it is better to be defined and used with `useTranslation`

Reply:
This alert message was hardcoded for an emergent fix, and can be removed now. The `alert message` was mistakenly thrown from the node, so it's filtered out in the frontend. Now the message had been removed from ckb node

Reply:
done